src/master/master.js

```diff
@@ -138,10 +179,17 @@ export class Master {
       deltalog: undefined,
     };
 
+    const log = redactLog(
+      this.game.flow.redactedMoves,
+      state.deltalog,
```

Comment:
Shouldn't this be `state.log` here?

Reply:
Hah. Tests don't care whether to use `state.log` or `state.deltalog` here. Both are `undefined` in all current test cases.

Reply:
Yeah agree, `sync` should care abouth the whole log, whereas `update` should care about the delta one.